packages/material-ui/package.json

```diff
@@ -41,6 +41,7 @@
     "@types/react-transition-group": "^2.0.8",
     "brcast": "^3.0.1",
     "classnames": "^2.2.5",
+    "css-mediaquery": "^0.1.2",
```

Comment:
Like for `recompose` we should not be targeting semver MINOR for 0.x versions. Package was last updated 5 years ago and looks stale. Looks like this is only used for SSR so I would rather let people inject a custom query method for SSR.

Reply:
> We use it for partial old browser support too

[window.matchMedia support](https://caniuse.com/#feat=matchmedia) matches our `.browserlistrc`. Why do we need to extend the browser support here?

> The package was written by https://twitter.com/ericf, not some random guy

I was not talking about the author but rather the state of the package overall. The semver concerns are still valid though.

Reply:
Continuing this conversation in `unstable_useMediaQuery.js`